multiversx_sdk_cli/tests/test_proxy.py

```diff
@@ -55,7 +56,8 @@ def test_get_hyperblock_by_hash():
 
 
 def test_sync_nonce():
-    account = Account("erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th")
+    account = Account()
```

Comment:
why not `account = Account(address = Address.from_bech32("erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th"))` ?

Reply:
Done! Also the test is a bit "fragile" and will soon fail after the reset of the devnet.